pkgs/top-level/python-packages.nix

```diff
@@ -17343,7 +17343,8 @@ in {
 
     propagatedBuildInputs = with self; [ pkgs.syncthing dateutil pyinotify
       pkgs.libnotify
-      (pkgs.librsvg.override { withGTK = true; })
+      (pkgs.librsvg.override { withGTK = true; enableIntrospection = true; })
```

Comment:
Actually, `withGtk` implies `enableIntrospection`. But in this case , the latter should be enough.

Reply:
Oh, right, it no longer does since https://github.com/NixOS/nixpkgs/issues/32296. But enabling introspection should be still enough.